### PR TITLE
Ipb 524/task list page

### PR DESCRIPTION
**Pull request by kth13**
## What does this pull request do?

- Newly designed Task list page
- showing the PoP name and CRN in every page
- removing the blue banner until the last page
- updating capturing the location page

## What is the intent behind these changes?

- we need to capture different informations based on different type of referrals

### CODE REVIEW

**Review by marcocolluramoj — APPROVED** [review]

File: server/app.ts
Comment (awhodgson256) [inline server/app.ts:1]: Not sure what this is for?  Is it some sort of dependency import?

Comment (rgforsyth) [reply server/app.ts:1]: ### allows inline scripts to run. Often we've had to add to here when adding the auto-complete box

File: server/routes/makeAReferral/confirm-probation-practitioner-details/confirmProbationPractitionerDetailsPresenter.ts
Comment (awhodgson256) [inline server/routes/makeAReferral/confirm-probation-practitioner-details/confirmProbationPractitionerDetailsPresenter.ts:1]: Don't we have an existing submission at /form endpoint?

File: server/app.ts
Comment (awhodgson256) [reply server/app.ts:1]: OK no worries

File: server/routes/makeAReferral/confirm-probation-practitioner-details/confirmProbationPractitionerDetailsPresenter.ts
Comment (kth13) [reply server/routes/makeAReferral/confirm-probation-practitioner-details/confirmProbationPractitionerDetailsPresenter.ts:1]: @awhodgson256 The task list design page got changed and after confirm probation practitioner page, we go back to the form page

**Review by rgforsyth — APPROVED** [review]

File: server/routes/makeAReferral/confirm-probation-practitioner-details/confirmProbationPractitionerDetailsPresenter.ts
Comment (awhodgson256) [reply server/routes/makeAReferral/confirm-probation-practitioner-details/confirmProbationPractitionerDetailsPresenter.ts:1]: Thanks for clarifying.  All good then, approving change now.

**Pull request merged**